### DISCUSSION TITLE
SCP-166 - 只是一位少女盖亚

### DISCUSSION THREAD

**Discussion by 1ctrl-cv** [share121/inter-knot#3465] [其他]
> “[SCP-166](https://scp-wiki-cn.wikidot.com/scp-166)”，作者 Ross Fisher-Davis，重写者 DrClef 与 Cerastes，来自 [SCP 维基](https://scp-wiki.wikidot.com/)。原文链接：https://scp-wiki.wikidot.com/scp-166 。译者 Ground0 与 Eurya，来自 [SCP-CN 维基](https://scp-wiki-cn.wikidot.com/)。译文链接：http://scp-wiki-cn.wikidot.com/scp-166 。遵循 [CC-BY-SA](https://creativecommons.org/licenses/by-sa/3.0/deed.zh) 协议。

**项目编号：** SCP-166

**项目等级：** Euclid

**特殊收容措施：** SCP-166被收容于Site-19的生物收容区C，该区域已经过改造，包括一个密闭的前厅及一台工业级空气净化器。当处于SCP-166的收容区域内时，收容人员必须随时穿着特殊设计的166生物危害防护服。

由于SCP-166的独特生理需求，已提供各式宽松的有机棉服装，供每月循环使用。所有食物均需按提供的指导烹饪，并尽可能少地使用无机添加剂。

获得个人用品及改进收容套间的合理请求可在--4级或更高级别人员--同意后批准。*更新：SCP-166的所有请求必须由站点主管Light亲自批准。* 目前为止，SCP-166已请求：

* 一本圣经（Douay-Rheims版，Challoner订本）（被批准）。
* 一条天主教玫瑰念珠（被批准）。
* 拜访一位天主教神父以进行忏悔、弥撒以及其它的圣礼。~~（被拒绝）~~（Davis牧师已被安排在每两周的星期天与SCP-166会面，会面前需彻底清除污染）
* 各种书刊杂志，大部分是宗教性的（批准，需对内容进行审核）。
* 一台电话，用来联系爱尔兰戈尔韦郡仁慈圣母女修道院的院长 ~~（被拒绝）~~ ~~（被批准）~~（在站点主管的命令下驳回，被拒绝）

**描述：** SCP-166是一名处于青少年晚期的欧裔女性人类，具有蹄类动物特征；拥有鹿角、蹄和短尾，令人联想起*Rangifer tarandus*（普通驯鹿）。但除了这些明显的异常之外，DNA分析并未发现异常的遗传特征。

在SCP-166周围半径15米范围内，人造物品逐渐恢复到未加工状态。较复杂的物品，如电子产品或车辆会更快受到影响，因为金属元件的降解会在几小时内造成毁灭性的结构故障。用基础材料制成的物品，如石头建筑或有机材料产品，其衰变速度几乎无法察觉。在相同的半径范围内，植物将开始发芽，通常在不太可能的地方生长，如无安全摄像头或ID扫描仪处。

SCP-166对人工材料及污染物可能异常性地敏感，吸入或接触可引起急性哮喘发作和压疮等症状。一个案例中，与吸烟者的身体接触导致SCP-166经历了严重的哮喘发作，尽管当时那位博士已经三周没有吸烟。

**发现：** SCP-166被发现于爱尔兰戈尔韦郡的慈悲圣母修道院，它自婴儿期开始就生活在那里。SCP-166被一名叛变的全球超自然联盟特工确认为威胁实体9927-Black（“女神”），或称SCP-████的孩子，它在所谓的康沃尔事件中被一支GOC攻击小队处决。
\
\
**恢复的GOC文件**

**威胁实体数据库条目**

**威胁ID：**
**KTE-9927-Blackchild “女儿”**

**授权应对等级：**
**4（严重威胁）**

**描述：**

威胁实体为具现化的LTE-9927-Black（“女神”）与未知父亲的孩子。尽管与它母亲极为相似并拥有其动物特征，实体不像9927-Black那样有极类野兽的外貌。有轻微的植物驱动能力，但对威胁实体做如此分类的原因主要是它对超自然程序时钟黑之子哈腓拉天生的了解及执行能力，这是一种世界范围内的仪式，可以将人类文明不可逆地倒回新石器时代阶段。

攻击小队“兰斯洛特”于19██年在后来被称作臭名昭著的“康沃尔事件”的行动中，在英国使9927-Black无效化，但由于其队长Ukulele特工的死亡，未能证实9927-Blackchild是否已被清理。Ukulele死后被授予Silver Aegis嘉奖，以表彰他对人类的终身贡献。

**清理：**
威胁实体未采取任何防卫手段。以压倒性的伤害完成处决。
\
\
\
特工拒绝处决SCP-166，而将其偷偷带到爱尔兰戈尔韦郡的一个天主教修道院。它12岁前一直住在那里，直到一位修道院的访客偶然目击了SCP-166，并向有关部门报告此事。这位特工随即联系基金会，同意分享GOC的情报，以换取SCP-166的安全和收容。

进一步的细节属于机密。
\
\
**附录166.1：** Davis牧师每两周的会面

**Davis：** 早上好，孩子。

**SCP-166：** 早上好，神父。

**Davis：** 照例，我得提醒你，由于我们所处的环境，告解内容不会保密，除非特殊要求。即使如此，如果被认为有必要的话，我们的谈话细节依然会被公开。明白吗？

*SCP-166点点头。*

**Davis：** 很好。那么，你最近怎么样？

**SCP-166：** 挺好的。有一个员工昨天跟我讲了Benedict的事，那是真的吗？

**Davis：** 啊，是的，那非常不幸，但也可以理解。他就职时已经很老了。知道他为教会尽心尽责，现在他可以休息了。

**SCP-166：** 你知道谁会接替他吗？

**Davis：** 有很多猜测，但任何人都有可能。毕竟有着最近所有这些……争端，现在是一段困难时期。他们可能想要一个新面孔来代表教会，或者他们会选择一个已经奉献多年的人。谁知道呢，他们甚至可能会挑出一个工人。这肯定会给人们带来一些谈资。

**SCP-166：** 我猜也是。

*SCP-166和Davis陷入沉默。*

**Davis：** 我感觉你还有问题想问，孩子。

**SCP-166：** 对不起。

**Davis：** 没必要道歉。不管怎么说，这就是我来这里的目的。你要问什么？

**SCP-166：** 我只是，只是想问你些事，我想可能有点私人。我只是想知道，你和你的父母关系好吗？

**Davis：** 对于我的母亲，是的。在她去世前，我每个月去养老院看望她一次，她生日和假期的时候也去。我告诉她，我是一个随军牧师，我想这还比较接近事实。

**SCP-166：** 那你的父亲呢？

**Davis：** 这个问题比较复杂。他是一个好人，一个紧握三样事物不放的士兵：上帝、国家和家庭。不幸的是，他非常严苛地坚持这些信仰，这导致了一些……激烈的讨论。我依然爱他，但这样对所有人都好。

*Davis叹了口气。*

**Davis：** 那你的父母呢？我知道你生活在修道院里，但在那之前呢？

**SCP-166：** 我从没有真正了解过他们，在我被遗弃时我还是个婴儿。我是说，如果他们把我放在那儿的话，他们一定认识那些修女，但我不记得了。我只是零星知道一些。他们说了一些有关我母亲的事，但接着就意识到他们谈论我时应该注意点。我想他们说她是一个女神？很明显那不可能是真的，她不可能是某种灵体，一定是什么东西，如果我最终长成这样的话。

*SCP-166指向她自己。*

**SCP-166：** 我记得偷听过院长的谈话，她正在对另一个修女说，她做错事了，是一件有关被其他人阻止的仪式的事。他们说她死了。

**Davis：** 很抱歉听到你母亲去世的事。

**SCP-166：** 不像我知道她那样。

**Davis：** 那你的父亲呢？

*SCP-166犹豫了。*

**SCP-166：** 我不知道。他一定是把我遗弃在修道院的那个人，但为什么是在那儿呢？为什么他不带着我和他一起？

**Davis：** 我相信他有自己的原因。

**SCP-166：** 或许吧。你知道，他们从来没谈论过他。一次也没有。我肯定已经问过院长有一千遍了，但她甚至从没有提起过一点点有关他的事。

*SCP-166停顿了一下。*

**SCP-166：** 如果我的母亲这么可怕……那么我的父亲做了什么呢？

**[日志结束]**
\
\
\
\
**附录166.2：** 对[已编辑]的纪律谈话
\
**[日志开始]**

**Light：** 你当时 *他妈* 在想什么？

**[已编辑]：** 我想保证她平安无事。你们不让我和她说话。我选了另一条路。

**Light：** 你做的事比那更糟糕。如果你只是为了她的舒适而滥用职权，当然，我可以睁一只眼闭一只眼。但你却开始试图给一个4级异常提供通到外界的电话——该死。委员会已经对你和她在同一个地方工作感到厌恶了，这件事已经暴露，你可以和你做的所有交易说再见了。

**[已编辑]：** 别这样啊，Sophia！她 *不会伤害别人。* 她待在这儿的唯一原因就是我。我必须做些什么。基金会会就这样让她长大吗，想想她——

**Light：** 在你把另一个词说出来之前，记住，这将对所有获得四级许可的人公开。我可以编辑你的名字，但我不能阻止人们在一次不合时宜的突发事件中把这些信息拼凑起来。

*[已编辑]保持沉默。*

**[已编辑]：** 十六年了。十六年里，她不能在城市里走动，不能去看电影，甚至去购物也不能做到。不管是在修道院或是在基金会的单间里，她都因为她无法选择的事被关了起来。所有这些都是因为我。这不公平。

**Light：** 我知道。

**[已编辑]：** 而我对此无能为力。我可以把突击队[^1]派遣到世界上的任何地方，我知道世上最有权势的人会花数十亿收买的秘密，但我甚至不能和她说话，让她知道自己不孤单。

**Light：** 你已经尽你所能了。在这样一个棘手的处境中，你表现得比任何人所能期盼的都要好。

**[已编辑]：** 真滑稽啊，那几乎一点用也没有。我——

*[已编辑]陷入沉默。*

**[已编辑]：** 你知道，我不在乎。把我做的都记下来吧。让我们快点结束这一切。

**Light:** ……我给你安排了六节与基金会心理学家的会面，每次2小时，我保证是和Glass的。他在最后签名了，我们可以把这从你的记录里抹消掉。

**[已编辑]：** 哦。

**Light：** [已编辑]。

**[已编辑]：** 什么？

**Light：** ……

**[已编辑]：** ……好，我明白了。谢谢你，Sophia。

**[日志结束]**
\
\
\
\
2013年5月8日，以下信件被发现于SCP-166的收容区域内。

> ████，
> ~~~~
> 我遇到你母亲的时候，我们都还是刚长大的孩子。她的脚上长着蹄子，眼中却闪烁着星光。她是那么美，那么天真，而我却亲手杀害了她。
> ~~~~
> 伊甸园不是一个地方。那是一种存在的状态。他们想要让我们回到那样，我阻止了他们。我又一次把天堂从我们这里夺走了。我从未对我那一天所做的事情感到悔恨，除了一件事：当你那天第一次见到我的时候，你看到你的父亲把一发子弹射进了你母亲的头里。我没有借口，没有解释。你甚至可能不记得了，但我现在对你说了这件事，希望你能理解我所做的一切。我希望你宽恕我。
> ~~~~
> 我爱你。我希望我本可以为你做的更多。我能为你所做的最好的事情是把你留给那些善良的人们，并希望她们能够替我抚养你。从我所见到的来看，她们做得很好。我很抱歉你不能和她们待在一起。我很抱歉她们把你送来这里。我发誓会尽我所能让你在这里的生活快乐。我发誓让你安全。
> ~~~~
> 十六岁生日快乐，
> [爱你的父亲。](https://scp-wiki-cn.wikidot.com/the-tombstone-of-alto-clef)  

\
\
\
\
**Footnote:**
[^1]: **译注：** 原文Strike team，是GOC攻击小队的原称。在此处应为其字面意思突击队，即可指联盟攻击小队，也有可能指基金会机动特遣队，毕竟此时文中的[已编辑]已经归属基金会了。

**Comment by Cross-king** [comment]
SCP绳网分部是吧

**Comment by lllliufeng** [comment]
出个SCP-IK（interknot）系列吧（

**Reply by 1ctrl-cv** [reply]
没人写文章啊（
